runtime/ftplugin/rasi.vim

```diff
@@ -0,0 +1,25 @@
+" Vim filetype plugin file
+" Language:		RASI
+" Maintainer:		Pierrick Guillaume <pierguill@gmail.com>
+" Last Change:		2024 May 21
+
+if exists("b:did_ftplugin")
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+let b:undo_ftplugin = "setl com< cms< isk<"
+
+setlocal comments=s1:/*,mb:*,ex:*/ commentstring&
```

Comment:
no need to reset `commentstring` option, since you are setting it in the next line anyhow.